lincheck/src/main/java/org/jetbrains/kotlinx/lincheck/strategy/Strategy.java

```diff
@@ -70,6 +80,13 @@ public boolean needsTransformation() {
         return false;
     }
 
+    /**
+     * Check if Strategy allows resuming the coroutine.
+     */
+    public boolean canResumeCoroutine(int iThread) { return true; }
```

Comment:
why `true`?

Reply:
For example, in ```StressStrategy``` it should be true. I don't see a reason to have it false by default.